keyboards/bpiphany/pegasushoof/config.h

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2016 Daniel Svensson <dsvensson@gmail.com>
+Copyright 2017 Danny Nguyen <danny@hexwire.com>
```

Comment:
Shouldn't this stay the same?

```suggestion
Copyright 2016 Daniel Svensson <dsvensson@gmail.com>
```

Reply:
This header is from Danny, so maybe?

Reply:
I moved the original file into the relevant 2013/2014 folder, this is copied from the iris for top level.